dotnet/src/SemanticKernel.Skills/Skills.OpenAPI/Model/RestApiOperationParameter.cs

```diff
@@ -12,6 +12,11 @@ internal class RestApiOperationParameter
     /// </summary>
     public string Name { get; }
 
+    /// <summary>
+    /// The parameter type.
```

Comment:
Can you add a couple examples of common types in the comment? "string" is one... what others might we expect?

Reply:
Done